docker/Dockerfile.cpu

```diff
@@ -15,6 +15,8 @@
 # Build arguments:
 #   PYTHON_VERSION=3.13|3.12 (default)|3.11|3.10
 #   VLLM_CPU_DISABLE_AVX512=false (default)|true
+#   VLLM_CPU_AVX2=false (default)|true|1 (for cross-compilation)
```

Comment:
Please also remove ```1``` to keep unified

And please help to add ```(for cross-compilation)``` to ```VLLM_CPU_AVX512BF16```, ```VLLM_CPU_AVX512VNNI```, ```VLLM_CPU_AMXBF16``` as well :)